tasks/server.yml

```diff
@@ -11,11 +11,20 @@
     group: "thumbor"
     system: "yes"
 
-- name: "Ensure thumbor application is present"
+- name: "Ensure latest version of thumbor application is present"
+  pip:
+    executable: "pip2"
```

Comment:
why pip2?

Reply:
beats me it was pip2 already, i left it as it were, chech the task below

Reply:
answered in external tooling.